test-yolo.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python
+# coding=utf-8
+
+#
+# Usage:
+#
+# Run all tests against PHP 7.3:
```

Comment:
Is this really a php 7.3 test, or is this comment left over? Maybe it should say "Drupal 8" instead?